### PR TITLE
hometask8

### PR DESCRIPTION
**Pull request by bogdanbasii**
The list of numbers validated as Ukrainian will be shown when command "ukr" is printed

### CODE REVIEW

File: hometask8.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+name = input('Enter a name:')
+
+number = input('Enter a number:')
+
+if phonebook.get(name):
+    print(f'{name} already exists. You need to delete a record and add again if you want to change it')
+else:
+    phonebook[name] = number
+    ukrphones = ''.join(re.findall("([+]?380\d{9}|0\d{9})", number))
```
Comment (klymenok) [inline hometask8.py]: Check for a number before you add it to a phone book. There shouldn't be non-ukrainian numbers in your phonebook

Comment (klymenok) [reply hometask8.py]: And your regular expression accept such invalid cases as:
```
809512345678
3809512345678912
phone380951234567

**Review by klymenok — CHANGES_REQUESTED** [review]

File: hometask8.py (hunk 1 above)
Comment (bogdanbasii) [reply hometask8.py]: I am already a bit confused with the code but I think I have fixed the issues

Comment (klymenok) [reply hometask8.py]: Your regex still allows these formats:
```
3809512345678912
phone380951234567

**Review by klymenok — CHANGES_REQUESTED** [review]
regex still need some changes

File: hometask8.py (hunk 1 above)
Comment (bogdanbasii) [reply hometask8.py]: There must be some mistake. My regex is "(^\W*[+]?380[\d]{9}|0[\d]{9}$)" and it doesn't allow such numbers

**Review by klymenok — APPROVED** [review]
Looks good!

**Pull request merged**